assets/src/application/ui/input/Button/Link.tsx

```diff
@@ -5,20 +5,30 @@ import { Tooltip } from '@infraUI/display';
 import { LinkProps } from './types';
 import './style.scss';
 
-const Link: React.FC<LinkProps> = ({ external, href, icon, tooltip, tooltipProps, ...props }) => {
+const Link: React.FC<LinkProps> = ({ children, external, href, icon, tooltip, tooltipProps, ...props }) => {
 	const className = classNames('ee-btn-base ee-icon-button ee-icon-button--borderless', props.className);
 
-	if (external && icon) {
+	if (external && !icon && tooltip) {
```

Comment:
my personal preference is avoiding nesting logic, this is why I didn't place this in the conditional below